lightning/src/ln/channel.rs

```diff
@@ -2028,7 +2043,7 @@ impl<Signer: Sign> Channel<Signer> {
 			if pending_remote_value_msat - msg.amount_msat - chan_reserve_msat < remote_fee_cost_incl_stuck_buffer_msat {
 				// Note that if the pending_forward_status is not updated here, then it's because we're already failing
 				// the HTLC, i.e. its status is already set to failing.
-				log_info!(logger, "Attempting to fail HTLC due to fee spike buffer violation");
+				log_info!(logger, "Attempting to fail HTLC due to fee spike buffer violation in channel {}. Rebalancing is required.", log_bytes!(self.channel_id()));
```

Comment:
Not a big deal but wish we could point people somewhere for expansion on "rebalancing is required" :thinking:

Reply:
We'll get there eventually :)  #959